docs/example-scenario/data/ecommerce-order-processing.md

```diff
@@ -0,0 +1,116 @@
+
+Azure Monitor provides unified user interfaces for monitoring across various Azure services. For more information, see [Monitoring in Microsoft Azure](/azure/monitoring-and-diagnostics/monitoring-overview). Event Hubs and Stream Analytics are both integrated with Azure Monitor. 
+
+For other availability considerations, see the [availability checklist][availability].
+
+### Scalability
+
+Kafka on HDInsight allows [configuration of storage and scalability](/azure/hdinsight/kafka/apache-kafka-scalability) for Kafka clusters. Cosmos DB provides fast, predictable performance and [scales seamlessly](/azure/cosmos-db/partition-data) as your application grows.
+The event sourcing microservices-based architecture of this scenario also makes it easier to scale your system and expand its functionality.
+
+For other scalability considerations, see the [scalability checklist][scalability] available in the Azure Architecture Center.
+
+### Security
+
+The [Cosmos DB security model](/azure/cosmos-db/secure-access-to-data) authenticates users and provides access to its data and resources. For more information, see [Cosmos DB database security](/en-us/azure/cosmos-db/database-security).
+
+For general guidance on designing secure solutions, see the [Azure Security Documentation][security].
+
+### Resiliency
+
+The event sourcing architecture and associated technologies in this example scenario make this scenario highly resilient when failures occur. For general guidance on designing resilient solutions, see [Designing resilient applications for Azure][resiliency].
+
+## Pricing
+
+To examine the cost of running this scenario, all of the services are pre-configured in the cost calculator.  To see how pricing would change for your particular scenario, change the appropriate variables to match your expected data volume. For this scenario, the example pricing includes only Cosmos DB and a Kafka cluster for processing events raised from the Cosmos DB Change Feed. Event processors and microservices for originating systems and other downstream systems are not included, and their cost is highly dependent on the quantity and scale of these services as well as the technologies chosen for implementing them.
+
+The currency of Azure Cosmos DB is the request unit (RU). With request units, you don't need to reserve read/write capacities or provision CPU, memory, and IOPS. Azure Cosmos DB supports various APIs that have different operations, ranging from simple reads and writes to complex graph queries. Because not all requests are equal, requests are assigned a normalized quantity of request units based on the amount of computation required to serve the request. The number of request units required by your solution is dependent on data element size and the number of CRUD operations per second. For more information, see [Request units in Azure Cosmos DB](/azure/cosmos-db/request-units). These estimated prices are based on Cosmos DB running in two Azure regions.
```

Comment:
Instead of "CRUD operations per second" I would say "number of database read and write operations"

Reply:
updated